static/js/ui.js

```diff
@@ -117,12 +117,7 @@ function message_hover(message_row) {
     message = current_msg_list.get(rows.id(message_row));
     message_unhover();
     message_row.addClass('message_hovered');
-    if ((message_edit.get_editability(message) === message_edit.editability_types.FULL) &&
-        !message.status_message) {
-        message_row.find(".edit_content").html('<i class="icon-vector-pencil edit_content_button"></i>');
-    } else {
-        message_row.find(".edit_content").html('<i class="icon-vector-file-text-alt edit_content_button" data-msgid="' + id + '"></i>');
-    }
+    message_row.find(".reaction_button").html('<i class="icon-vector-smile edit_content_button" data-msgid="' + id + '"></i>');
```

Comment:
So, we probably still want it to show the pencil for your own, editable messages; it's just for other users' messages (i.e. the `file` icon case) where we want to have the reaction icon.